File: EXERCÍCIOS JAVA E JAVA POO/Aula11_POO/src/aula11_poo/Aluno.java
package aula11_poo;
public /*final*/class Aluno extends Pessoa {
    //final indica que a classe é a última não podendo ter herdeiros(extends)
    private int matricula;
    private String curso;
    
    public /*final*/void pagarMensalidade(){
        //final indica que não pode haver sobreposição
        System.out.println("Pagando mensalidade de aluno "+this.nome);
    }

    public int getMatricula() {
        return matricula;
    }

    public void setMatricula(int matricula) {
        this.matricula = matricula;
    }

    public String getCurso() {
        return curso;
    }

    public void setCurso(String curso) {
        this.curso = curso;
    }
}

File: CURSO PHP POO/CURSO POO PHP/Aula11/Aluno.php
<?php
require_once 'Pessoa.php';
/*Final*/class Aluno extends Pessoa{
    //Final não deixa essa classe ser herdada
    private $matricula;
    private $curso;
    
    public /*Final*/function pagarMensalidade() {
        //Final não deixa ser sobreposto
        echo "<p>Pagando Mensalidade do aluno </p>".$this->nome;
    }
    function getMatricula() {
        return $this->matricula;
    }

    function getCurso() {
        return $this->curso;
    }

    function setMatricula($matricula) {
        $this->matricula = $matricula;
    }

    function setCurso($curso) {
        $this->curso = $curso;
    }


}
